README.md

```diff
@@ -14,64 +15,83 @@ Please see here for more details:
 * https://members.openpowerfoundation.org/wg/ACLWG/dashboard
 * http://openpowerfoundation.org/blogs/capi-drives-business-performance/
 
-For detailed design information, please refer to the SNAP Workbook (available soon).
-
-# Getting started
-
-## Simulating the design and generating the bitstream
-
-The resources for generating a simulation model and an FPGA image using the SNAP framework are located in the [hardware](hardware) subdirectory of this repository. For information on how to use them please refer to the documentation in the
+## What is CAPI, education materials and more information
+* CAPI and SNAP on IBM developerworks: https://developer.ibm.com/linuxonpower/capi/  
+* [Education Videos](https://developer.ibm.com/linuxonpower/capi/education/)
+* [IBM CAPI Developer's Community Forum](https://www.ibm.com/developerworks/community/groups/service/html/communitystart?communityUuid=a661532e-1ec6-442f-b753-4ebb2c8f861b)
 
-* [README.md](hardware/README.md)
+## Status
+Currently SNAP Framework supports CAPI1.0. The modules for CAPI2.0 and OpenCAPI are under developing. However, users working on SNAP today can easily transfer their work to CAPI2.0 or OpenCAPI as the interface for "**Software Program**" and "**Hardware Action**" (shown in yellow area of the above figure) will keep same. 
 
-file within that directory.
 
-## Flashing the bitstream
+# 2. Getting started
 
-Please see [Bitstream_flashing.md](hardware/doc/Bitstream_flashing.md) for instructions on how to update the FPGA bitstream.
+Developing an FPGA accelerated application on SNAP takes following steps: 
+* **Preparation**: Decide the software function to be moved to FPGA. This function, usually computation intensive, is named as "action" in the following description. Carefully read the **Dependencies** in following section. Choose an FPGA card. Install tools, download packages and set environmental variables.
 
```

Comment:
I believe we shouldn't "enforce" these steps and annoy the experts. We can propose this as one way, but experienced FPGA developers may simply jump right into an existing example and tweak it to their needs.